KristenMcWilliam.Nyrna version 2.1.0

Merrit opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25052)

Merrit committed:
KristenMcWilliam.Nyrna version 2.1.0

ImJoakim reviewed (changes requested):
The DisplayVersion is `2.0.0-beta.2`. If you want to upload this specific package, you would need to change the version to that. However, I see that you're the person who released it, so maybe you are able to update the DisplayVersion to `2.1.0` (which I'm guessing it's supposed to be) and you won't have to call it `2.0.0-beta.2`.

![image](https://user-images.githubusercontent.com/52826317/129458774-6bfe4652-f97f-4dcc-b9ec-8dc8a4d7fd3e.png)

ImJoakim commented:
And the DisplayName:

![image](https://user-images.githubusercontent.com/52826317/129458830-f83fc77e-213c-4c18-b0f9-4027a2a0c7ea.png)

Merrit commented:
@ImJoakim Nice catch, thank you!

I've updated my CI pipeline to properly forward the current version info for both variables to inno setup, and updated the binary at the download url.

ImJoakim commented on manifests/k/KristenMcWilliam/Nyrna/2.1.0/KristenMcWilliam.Nyrna.installer.yaml:
```diff
@@ -0,0 +1,13 @@
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/Merrit/nyrna/releases/download/v2.1.0/nyrna-windows-installer.exe
+  InstallerSha256: 5F11F3B979E346CE57D27B06AEE3D1BD435D6EE6C72ADB754DE562AEA1B240C4
```
```suggestion
  InstallerSha256: 7672C8659AABDF5D9A54F61BAEF2EFACC1C5AE7459B78DE7FA972896F48353B2
```

ImJoakim reviewed (changes requested):
New hash: https://github.com/microsoft/winget-pkgs/pull/25052#discussion_r688992066

Merrit committed:
Update manifests/k/KristenMcWilliam/Nyrna/2.1.0/KristenMcWilliam.Nyrna.installer.yaml

Co-authored-by: Joakim <52826317+ImJoakim@users.noreply.github.com>

ImJoakim commented:
Perfect 😄 

![image](https://user-images.githubusercontent.com/52826317/129460232-8cb259e7-2271-4560-9967-8ec90ce819c4.png)

Merrit commented:
Awesome, much appreciate your help! 😀
